R4ML/R/zzz.R

```diff
@@ -355,7 +363,12 @@ r4ml.session.stop <- function() {
   # clean sparkR
   sparkr.stop <- function() {
      if("sc" %in% ls(.GlobalEnv)) { rm(sc, envir = .GlobalEnv) }
-     SparkR::sparkR.session.stop()
+     # only remove if r4ml started the sparkr session
+     if (identical(SparkR:::.sparkREnv, r4ml.env$sparkr.env)) { 
```

Comment:
I think, we can't as they are two different tools so it needs to have it's own init.